<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetWindow</class>
 <widget class="QMainWindow" name="SetWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>623</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>706</width>
    <height>623</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>706</width>
    <height>623</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>421</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>初始资金</string>
    </property>
    <widget class="QLineEdit" name="moneyText">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>421</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>币种</string>
    </property>
    <widget class="QRadioButton" name="type1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>人民币</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="type2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>美元</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="type3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>欧元</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="type4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>英镑</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>421</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>支付方式</string>
    </property>
    <widget class="QRadioButton" name="pay1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>支付宝</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="pay2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>微信</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="pay3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>银行卡</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>440</y>
      <width>431</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>牌数</string>
    </property>
    <widget class="QRadioButton" name="card1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="card2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="card3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>530</y>
      <width>251</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>进入游戏</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>531</width>
      <height>101</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>421</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>玩家人数</string>
     </property>
     <widget class="QRadioButton" name="playernumber_1">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>115</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="playernumber_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>115</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="playernumber_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>115</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>706</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>游戏设定</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
